src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -158,11 +158,11 @@ def test_fetch_multiple(single_channel_session):
 def test_measure_multiple(session):
     with session.initiate():
         # session is open to all 12 channels on the device
-        voltage_measurements, current_measurements = session.measure_multiple()
-        assert len(voltage_measurements) == len(current_measurements) == 12
+        measurements = session.measure_multiple()
+        assert len(measurements) == 12
```

Comment:
should also test that the tuple has voltage / current members.

Reply:
Added